pkgs/games/quake3/quake3e/default.nix

```diff
@@ -15,7 +15,11 @@
 , copyDesktopItems
 , makeDesktopItem
 }:
-
+let
+  arch =
+    /**/ if stdenv.hostPlatform.isx86_64 then "x64"
```

Comment:
Why the `/**/`?

Reply:
Just a formatting thing that's used a lot in nixpkgs with if statements.

Reply:
Ah, seems so. Okay then